internal/commands/app/create_inputs.go

```diff
@@ -88,29 +85,28 @@ func (i *createInputs) resolveDirectory(wd string) (string, error) {
 		return "", appErr
 	}
 	if appOK {
-		return "", fmt.Errorf("%s is inside or is a Realm app directory", dir)
+		return "", fmt.Errorf("%s is inside or is a Realm app directory", fullPath)
```

Comment:
ideally we use some form of `errProjectExists` here, so that its consistent b/w `app init` and `app create`